source/fundamentals/authentication.txt

```diff
@@ -1 +1,208 @@
+see the corresponding syntax for specifying the default authentication mechanism:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: default-connection-string
+
+      .. code-block:: csharp
+
+         var mongoClient = new MongoClient("mongodb://<username>:<password>@<hostname>:<port>/?authSource=<authenticationDb>");
+
+   .. tab:: MongoCredential
+      :tabid: default-mongo-credential
+
+      .. code-block:: csharp
+
+         var credentials = MongoCredential.CreateCredential("<authenticationDb>", "<username>", "<password>");
+         var settings = MongoClientSettings.FromConnectionString("<host>");
+         settings.Credential = credentials;
+         var mongoClient = new MongoClient(settings);
+
+.. _csharp-scram-sha-256:
+
+SCRAM-SHA-256
+~~~~~~~~~~~~~
+
+``SCRAM-SHA-256`` is a salted challenge-response authentication mechanism (SCRAM)
+that uses your username and password, encrypted with the ``SHA-256`` algorithm,
+to authenticate your user. 
+
+.. tip:: Default Mechanism
+
+   MongoDB version 4.0 and later uses SCRAM as the default authentication mechanism.
+   To learn more on specifying the default mechanism, see :ref:`csharp-authentication-default`.
+
+.. _csharp-scram-sha-1:
+
+SCRAM-SHA-1
+~~~~~~~~~~~
+
+``SCRAM-SHA-1`` is s a salted challenge-response mechanism (SCRAM) that uses 
+your username and password, encrypted with the ``SHA-1`` algorithm, to authenticate 
+your user.
+
+.. tip:: Default Mechanism
+
+   MongoDB version 4.0 and later uses SCRAM as the default authentication mechanism.
+   To learn more on specifying the default mechanism, see :ref:`csharp-authentication-default`.
+
+.. _csharp-x509:
+
+X.509
+~~~~~
+
+The ``X.509`` authentication mechanism uses `TLS <https://en.wikipedia.org/wiki/Transport_Layer_Security>`__
+with ``X.509`` certificates to authenticate your user, identified by the
+distinguished names of your client certificate. When you specify the 
+``X.509`` authentication mechanism, the server authenticates the connection using 
+the subject name of the client certificate.
+
+To learn more about using TLS/SSL, see our :ref:`TLS/SSL guide <csharp-tls>`.
+
+To learn more about ``X.509`` certificates, see the :ref:`X.509 guide <x509-client-authentication>`.
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tab to
+see the corresponding syntax for specifying the ``X.509`` authentication mechanism:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: default-connection-string
+
+      .. code-block:: csharp
+
+         var connectionString = "mongodb://"<hostname>"/?authMechanism=MONGODB-X509";
+         var settings = MongoClientSettings.FromConnectionString(connectionString);
+
+         settings.useTls = true;
+         settings.SslSettings = new SslSettings
+         {
+            ClientCertificates = new List<X509Certificate>()
+            {
+                new X509Certificate2("<path to X.509 certificate>", "<X.509 certificate password>")
```

Comment:
same thing here-- if the <> phrases are placeholders, do you need the quotes?

Reply:
In this case I think the quotes are helpful as they tell the user that the placeholder is meant to be a string value rather than some kind of object or something. Since it doesn't seem to be effecting the rendering in this case I'd vote to leave them in here.